third-culture-software/bhima
# Release version v1.17.0

🚀 Let's release bhima (currently at 1.16.1)


Changelog:
* fix(feeCenters): fix name collision with report (2bef721b0)
* docs: add documentation on debtor groups (3c8c9436b)
* improvement: add more whitespace above and below the grid footer template (e9510c2b2)
* fix(reports): fix client support error coloration (fcefa1fe3)
* Improvement: Update color css property (bd7cc4045)
* improvement: Using HEX8 for the colors for Highlighting the rows transparency and HEX6 for color of icon for legend (435ddc2e5)
* improvement(Stocks lots page) - Add legend for the colors of the rows in the stock lots page (1e2937e25)
* fix(credit note): do not reload entire registry (2e932133a)
* Renamed various report names to use underscores (a16f719f6)
* improvement(Dashboard stock) - Remove params transclude - Use separate small components for all panel - Remove component bhDashboardstock (b725b0bd1)
* Feature: Use new component for stock expired (fa21eadeb)
* improvement: using HEX color instead HEX8 (1ee754b5f)
* Clean and sanitaze code (40cc81726)
* improvement(DashbordStock) - Clean and sanitaze code (eef9d832a)
* Fix and solve conflict (61f1cfe5f)
* Fix and solve conflict with branch master (fcccf560d)
* Fix and resolve conflict (1de1b17a2)
* Fix and resolve conflict (09b3b98a2)
* Solve conflict (181cb528d)
* bug(Stock Exit) - Addition in the function reinit() of the call of the function   loadInventories for a depot to restore to reload the products available for the depot (4afd78da8)
* 5126 - Fix french typ in cashbox (7bccfc2fa)
* Fix missing/extra translation tokens (67852cd95)
* Update bors.toml (6114ecebc)
* Minor improvements to migration SQL script (ee4b91a60)
* fix(bhFindEmployee): make search work (55898f82b)
* feat(stock): implement stock changes report (f4493908a)
* refactor(stock): improve find modals (a02f94b84)
* test(Improvement) - calling .toDate in moment objetc (5ed3061f6)
* Improvement: delete .only in intergation test for purchase (0559eb8a2)
* use of configured algo for cmm (6a1fbd3f1)
* Test: Fix and solve test inventory adjustement (770ca13c6)
* test: Reset value for test (84a0cfec5)
* Test(Use Date Dynamically) - Use of dynamic dates - Skip csv file import tests - Modification of the values of the test results² (8b8bfb615)
* Add the date (d3bd76621)
* 4834 - Fix calculation and status about expired and at risk lots (4da5bd178)
* Fixed French translation per Bruce suggestion (c12dd4c57)
* Added PR number (b3c30b1be)
* Fix typo report name 'System Usage Statistics' (2cdb6da09)
* Fixed bugs in editing of user roles (9858964d7)
* Updated with suggestions from @jniles (84624e019)
* Add message for client on errors (ef4dda79a)
* improvement(Break Even Report) - Eliminate whitespace and not necessary <hr> - Remove property bold on checkboxes and radios buttons (f25db137d)
* Add user documentation for stock settings page. (85ef08c2d)
* enhance the check for validity of data (73fa552c5)
* Force formatting for stock import (c636baea5)
* Delete .doclets.yml (9e6206d0c)
* fix(stock): make adjustments one HTTP request (28d7d8536)
* feat(stock): receipt component stock movements (85791f0e3)
* bug(Out Stock) - Use of another variable to check if the repository has inventories in   stock for distribution instead of the vm.selectableInventories variable   which is used in multiple times (a11036640)
* refactor(stock): entry from purchase (02d2c3642)
* feat(references): add "view in account report" (e7bdcd4ca)
* test: update plage date for purchase integration (dfaaa3b36)
* Test: Use Date Dynamically in Intergration test (6a2ae5ce3)
